docs/reference/commandline/save.md

```diff
@@ -12,7 +12,7 @@ parent = "smn_cli"
 
     Usage: docker save [OPTIONS] IMAGE [IMAGE...]
 
-    Save an image(s) to a tar archive (streamed to STDOUT by default)
+    Save images to a tar archive (streamed to STDOUT by default)
```

Comment:
I like the second suggestion to make it consistent with `docker rm`. I would like to avoid the parentheses-S form.